src/components/EditTrip.js

```diff
@@ -102,46 +101,46 @@ export default function EditTrip() {
       <div className="editTripHeader">
         <h2>EDIT TRIP</h2>
       </div>
-      <EditTripForm
+      <Form
         handleOnSubmit={handleOnSubmit}
+        handleSightseeingOnClick={handleSightseeingOnClick}
+        handleExpenseOnClick={handleExpenseOnClick}
         inputDestinationName={inputDestinationName}
         setInputDestinationName={setInputDestinationName}
```

Comment:
I don't think you should be passing down the setState functions as props. But rather keep them up here, and then have the triggering event in the Form.js call a function in EditTrip.js, an insted pass the function down as a prop.

Not too sure, I am correct here, @mpagels maybe you can confirm?